fossunited/foss_hackathon/doctype/foss_hackathon_participant/foss_hackathon_participant.py

```diff
@@ -34,6 +34,7 @@ class FOSSHackathonParticipant(Document):
     def before_save(self):
         if self.has_value_changed("wants_to_attend_locally"):
             self.handle_localhost_request()
+
```

Comment:
We need to revert changes to this file. The changes are unrelated and don't need to be included in this PR.

```suggestion
```